Essentials/src/com/earth2me/essentials/commands/Commandwhois.java

```diff
@@ -30,38 +32,89 @@ public void run(final Server server, final CommandSource sender, final String co
         if (args.length < 1) {
             throw new NotEnoughArgumentsException();
         }
+        User player;
+        try {
+            final UUID uuid = UUID.fromString(args[0]);
+            player = ess.getUser(uuid);
+        }
+        catch (IllegalArgumentException ignored) {
```

Comment:
This code (and any that follow similarly) should be placed on the same line of code as the bracket to stay consistent with ess code style